chann.go

```diff
@@ -129,6 +130,7 @@ func New[T any](opts ...Opt) *Chann[T] {
 	case unbounded:
 		ch.in = make(chan T, 16)
 		ch.out = make(chan T, 16)
+		ch.backlog = make(chan T, 1024)
```

Comment:
Is the backlog being used somewhere?

Reply:
Oh this was outdated code, I have removed it.